docker-compose.yml

```diff
@@ -0,0 +1,108 @@
+version: '2'
```

Comment:
Second commit

Reply:
This is the [Compose File Version](https://docs.docker.com/compose/compose-file/compose-file-v2/). Version `2` is required instead of 3 since I haven't gotten around to fixing this error
```bash
Unsupported config option for services.kojo_fitness: 'volumes_from'
Unsupported config option for services.mysql: 'volumes_from'
Unsupported config option for services.pgsql: 'volumes_from'
```

Reply:
Ah, cool